apps/frontend/src/app/app.component.html

```diff
@@ -31,6 +31,6 @@ <h1>Welcome to Andrew's Super Amazing Music Taste!</h1>
 
 <footer>
   <small>
-    icons by <a target="_blank" href="https://www.svgrepo.com/">svgrepo</a>
+    icons by <a target="_blank" (neverGonnaGiveYouUp)="openKonami($event)" href="https://www.svgrepo.com/">svgrepo</a>
```

Comment:
😍

Reply:
i should put 2 directives, my logic is kinda meh